tests/conformance/data/NoData-pad-st-sum.expected.txt

```diff
@@ -0,0 +1,6 @@
+{"type":"Header","profile":"","library":""}
+{"type":"Statistics","messageCount":"BigInt(0)","channelCount":0,"attachmentCount":0,"chunkCount":0,"channelMessageCounts":{}}
+{"type":"SummaryOffset","groupOpcode":12,"groupStart":"BigInt(64)","groupLength":"BigInt(0)"}
```

Comment:
If we stick with JSON - maybe quoting the number values is enough (no need for BigInt wrapper). We can generate data with values in the expected numeric range to validate that each reader implementation is able to de-serialize.